app/src/main/java/org/mozilla/tv/firefox/generated/LocaleList.java

```diff
@@ -0,0 +1,9 @@
+package org.mozilla.tv.firefox.generated;
```

Comment:
I think we need to remove org.mozilla.focus.generated (I wonder why this gets generated into the src dir at all though...).